core/driver/sources/src/main/resources/reference.conf

```diff
@@ -0,0 +1,113 @@
+// Comments starting with the // signs are for descriptions and guides.
+probe {
+
+  // `intellij` is for org.virtuslab.ideprobe.config.IntellijConfig - default values below:
+  intellij {
+    plugins = []
+    version {
+      build = "212.5080.55"
+      release = "2021.2.1"
+    }
+    // Use below config with proper path if you want to use ane existing Intellij instead of downloading a version.
+    # path = ""
+  }
+
+  // `workspace` is for `Option[org.virtuslab.ideprobe.config.WorkspaceConfig]` - default None,
+  // hence comments added as a reference. You can specify the workspace by using below configurations.
+  # workspace {
+  // First option is to use a Default workspace - by just specifying `probe.workspace.path` with a valid string.
+  // `probe.workspace.path` can point to:
+  // a) a path on local filesystem (file or jar)
+  // or
+  // b) an online resource (http/https) - which can be for example a github repository
+  #   path = ""
+  //
+  // If `probe.workspace.path` is set and points to a git repository - you can choose one of three additional options:
+  // `branch`, `tag` or `commit` - to specify the reference which should be checked out on the workspace setup.
+  // Only one of them can be specified. You can as well not use any of these 3 options - in such scenario the default
+  // branch wii be checked out just like with a plain `git clone` command.
+  #   branch = ""
+  #   tag = ""
+  #   commit = ""
+  //
+  // Another option is to use an Existing workspace by passing a proper path to it's directory
+  // - use only `existing` config. In this case, `probe.workspace.path` MUST NOT be defined.
+  // `probe.workspace.existing` can be used for scenarios where you want to reuse an existing workspace instead of
+  // copying a workspace each time when tests run. It might be beneficial for big projects / repos.
+  #   existing = ""
+  # }
+
+  // `resolvers` is for `org.virtuslab.ideprobe.config.DependenciesConfig.Resolvers` - default values below
+  resolvers {
+    intellij {
+      repositories = []
+    }
+    plugins { }
+    jbr {
+      repositories = []
+    }
+    retries = 0
+  }
+
+  // `driver` is for `org.virtuslab.ideprobe.config.DriverConfig` - default values below
+  driver {
+    launch {
+      command = []
+      timeout = "30.seconds"
+    }
+    check {
+      errors {
+        enabled = false
+        includeMessages = [".*"]
+        excludeMessages = []
+      }
+      freezes {
+        enabled = false
+      }
+    }
+    xvfb {
+      screen {
+        width = 1920
+        height = 1080
+        depth = 24
+      }
+    }
+    headless = false
+    vmOptions = []
+    env = {}
+  }
+
+  // `paths` is for `org.virtuslab.ideprobe.config.PathsConfig` - default values below
+  paths {
+    // all parameters below are Options and are None by default - hence commented out
+    //
+    // `base` is the base directory used by ide-probe. All files and directories used by ide-probe in runtime
```

Comment:
Okay... but does it specifically mean that other keys from `paths` must point to subdirectories of `base`?